### PR TITLE
feat: plus page updates

### PR DESCRIPTION
**Pull request by dohaki**
Implements new designs for Across+ page https://www.figma.com/design/oISXonNeaMhANSbTdS6FZf/%E2%9D%8E-Website(s)?node-id=1929-11073&m=dev

### CODE REVIEW

File: src/app/(routes)/across-plus/_components/get-started-section-top.tsx
Comment (james-a-morris) [inline src/app/(routes)/across-plus/_components/get-started-section-top.tsx:28]: Did we want to add <br /> to the code?

File: src/app/(routes)/across-plus/_components/get-started-section-top.tsx
Comment (james-a-morris) [inline src/app/(routes)/across-plus/_components/get-started-section-top.tsx:11]: Should we use @/app here?

File: src/app/(routes)/across-plus/_components/mockups-carousel.tsx
Comment (james-a-morris) [inline src/app/(routes)/across-plus/_components/mockups-carousel.tsx:36]: OOC why do we not want to optimize this next/image tag?

**Review by james-a-morris — COMMENTED** [review]
Went through each file. There's some UI comments I had but I think Ryan raised them already

File: src/app/(routes)/across-plus/_components/get-started-section-top.tsx
Comment (dohaki) [reply src/app/(routes)/across-plus/_components/get-started-section-top.tsx:11]: These assets are not directly under `src/app/_assets` but rather in `src/app/(routes)/across-plus/_assets`. That's why I prefer `../` here because it's shorter

File: src/app/(routes)/across-plus/_components/mockups-carousel.tsx
Comment (dohaki) [reply src/app/(routes)/across-plus/_components/mockups-carousel.tsx:36]: I somehow can not get a crisp resolution to work without setting this flag to `true`... Any ideas?

File: src/app/(routes)/across-plus/_components/get-started-section-top.tsx
Comment (dohaki) [reply src/app/(routes)/across-plus/_components/get-started-section-top.tsx:28]: You mean the `<br />`? That is intentional yea

**Comment by dohaki** [comment]
> Went through each file. There's some UI comments I had but I think Ryan raised them already

Thanks! Yes, Ryan left a comment on the resolution of the mockup images

File: src/app/(routes)/across-plus/_components/get-started-section-top.tsx
Comment (james-a-morris) [reply src/app/(routes)/across-plus/_components/get-started-section-top.tsx:28]: Sorry the `<br />` actually split my comment. SGTM

File: src/app/(routes)/across-plus/_components/get-started-section-top.tsx
Comment (james-a-morris) [reply src/app/(routes)/across-plus/_components/get-started-section-top.tsx:11]: Ah gotcha - sgtm

File: src/app/(routes)/across-plus/_components/mockups-carousel.tsx
Comment (james-a-morris) [reply src/app/(routes)/across-plus/_components/mockups-carousel.tsx:36]: It's finicky especially since we're reducing the size of the image.

**Review by james-a-morris — APPROVED** [review]
LGTM pending final signoff from ryan

**Pull request merged**
